### DISCUSSION TITLE
Unable to import boiletplate

### DISCUSSION THREAD

**Discussion by Kiran-Serolla** [DevSkillsHQ/discusssions#7] [Help] [answered]
I am unable to import the boilerplate for the asignment. I get the following error at step2 :
export : The term 'export' is not recognized as the name of a cmdlet, function, script file, or 
operable program. Check the spelling of the name, or if a path was included, verify that the path  
is correct and try again.

**Comment by fenske** [comment, marked as answer]
Hi @Kiran-Serolla. I'm guessing you're on Windows. Try to substitute `export` with `set`. Does it work?

**Reply by Kiran-Serolla** [reply]
Thank you. And now I have one more basic question. Should I clone this repo: "https://github.com/93c6399/transaction-management-frontend-level-1_c9b32af-798851.git" that I got the invitation for or should I create my own repo and clone the boiler plate into it? I have been struggling to get started.

**Reply by fenske** [reply]
That's right. You should work on the repo you received in the invite and create a PR with your work in the end. That's how the hiring team will know you're done.

**Reply by Kiran-Serolla** [reply]
Thank you very much. I tried doing it.. the set thing worked but when I copied the code snippet , I get this error:  
At line:1 char:6
+ cat <<EOF >>init.sh
+      ~
Missing file specification after redirection operator.
At line:1 char:5
The '<' operator is reserved for future use.
At line:1 char:6
+ cat <<EOF >>init.sh.......  and so on..

**Comment by fenske** [comment]
I see. What I recommend, then is to import a boilerplate using an in-browser dev environment provided by Gitpod (it's Linux by default and has all you need pre-installed).

Just follow [this link](https://gitpod.io/#https://github.com/93c6399/transaction-management-frontend-level-1_c9b32af-798851) and it'll bootstrap the environment for you. (in this case, please use the original instructions for importing the boilerplate.

**Reply by Kiran-Serolla** [reply]
Thank you :)

**Reply by Kiran-Serolla** [reply]
Now I have another issue.. my changes are not reflected in the browser. I created a new component and added it to app-frontend/pages/index.js. But I dont see it on the browser/ output
also any edits I make on the existing component dosent reflect in output

**Reply by Kiran-Serolla** [reply]
I think I found solution for this
